docs/tldr/pages/linux/tmt.md

```diff
@@ -0,0 +1,36 @@
+# tmt
+
+> Test Management Tool for creating, running and debugging tests.
+> More information: <https://tmt.readthedocs.io>.
+
+- List available tests, plans, or stories:
+
+`tmt {test|plan|story} ls [<pattern>]`
+
+- Run all test steps for each plan:
+
+`tmt run`
+
+- Run selected plans and tests:
+
+`tmt run plan -n <plan_name> test -n <test_name>`
+
+- Show what tests a specific plan would run:
+
+`tmt run -vv discover plan -n <plan_name>`
+
+- Show test results from the last run in a web browser:
+
+`tmt run -l report --how html --open`
+
+- Add context to a test run:
+
+`tmt -c foo=bar -c baz=qux,quux run [...]`
+
+- Validate metadata:
+
+`tmt lint`
+
+- View documentation for tmt-try:
+
+`tldr tmt-try`
```

Comment:
How does the [`See also: \`command1\`, \`command2\``](https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md#subcommands) work from their documentation? I think we could combine the instructions for both `tmt run` and `tmt try` into one snippet?

Reply:
Well, I was following their example. +1 for "See also"

Reply:
Added